build.gradle

```diff
@@ -126,38 +148,69 @@ minecraft {
             }
         }
 
-        testClient {
+        gameTestClient {
             workingDirectory project.file('test-files/client')
             parent runs.client
 
+            def old = lazyTokens["minecraft_classpath"]
+            lazyToken("minecraft_classpath") {
+                // Add all files in testMinecraftLibrary to the classpath.
+                def allFiles = new HashSet<String>();
+
+                def oldVal = old?.get()
+                if (!oldVal != null && !oldVal.isEmpty()) allFiles.addAll(oldVal.split(File.pathSeparatorChar))
```

Comment:
what is `!oldVal != null`

Reply:
That's the minecraft classpath
The value of the returned supplier can be null, so we check that